add docs in reaction to aspnet/AspNetCore#8974 (#11945)
aspnet/AspNetCore.Docs: aspnetcore/signalr/configuration.md

    .build();
```

::: moniker range=">= aspnetcore-3.0"

> Instead of a `LogLevel` value, you can also provide a `string` value representing a log level name (see below for a list). This is useful when configuring SignalR logging in environments where you don't have access to the `LogLevel` constants.

Reviewer: Not sure "(see below for a list)" is needed. The article is short and you say the same thing in two sentences.

```javascript
let connection = new signalR.HubConnectionBuilder()
    .withUrl("/myhub")
    .configureLogging("warn")
    .build();
```

> The below table lists the available log levels. The value you provide to `configureLogging` sets the **minimum** log level that will be logged. Messages logged at this level, **or the levels listed below it on the table**, will be logged.

Reviewer 2: ["the following x" is preferred over the "the below x"](https://worldready.cloudapp.net/Styleguide/Read?id=2700&topicid=27393) because it sounds better when listening to a screen reader.

```suggestion
The following table lists the available log levels. The value you provide to `configureLogging` sets the **minimum** log level that will be logged. Messages logged at this level, **or the levels listed after it in the table**, will be logged.
```

> The below table lists the available log levels. The value you provide to `configureLogging` sets the **minimum** log level that will be logged. Messages logged at this level, **or lower levels on the table**, will be logged.

Reviewer 3: ```suggestion
The below table lists the available log levels. The value you provide to `configureLogging` sets the **minimum** log level that will be logged. Messages logged at this level, **or the levels listed below it on the table**, will be logged.
```

Otherwise, this could be misinterpreted as saying log levels with lower severity than the minimum log level will be logged which makes no sense.

> | `string` Name | `LogLevel` value |

Reviewer 2: Suggest removing code formatting from headings; with it they don't stand out as headings very well.  The headings could be just string and LogLevel.
```suggestion
| string | LogLevel |
```
